README.md

```diff
@@ -45,6 +45,22 @@ class PersonComponent {
 }
 ```
 
+### Example 4, using class method reference for onChange
+```typescript
+class PersonComponent {
+ 
+    @OnChange<string>('onNameChange')  
+    name: string;
+    
+    age: number;
+
+    onNameChange(this: PersonComponent, value, change) {
```

Comment:
We don't need type `this` anymore, so we can remove `this: PersonComponent`

Reply:
Removed the this parameter